client/src/components/SimpleDataField.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+const TextareaContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  background-color: ${(props) => props.theme.colors.input};
+  border-radius: 1.25rem;
+  border: none;
+  padding: 0.6rem;
+  outline: none;
+`;
+
+const Headline = styled.input`
+background-color: ${(props) => props.theme.colors.input};
+color: ${(props) => props.theme.colors.textSecondary}
+font-weight: bold;
+outline: none;
+cursor: pointer;
+max-width: 100%;
+border: none;
+border-bottom: dotted #707070;
+::placeholder {
+    color: ${(props) => props.theme.colors.textSecondary};
+`;
+
+const Textarea = styled.textarea`
+background-color: ${(props) => props.theme.colors.input};
+  color: ${(props) => props.theme.colors.textPrimary};
+  border: none;
+  outline: none;
+  cursor: pointer;
+  max-width: 100%;
+  resize: none;
+  wrap: "off";
+  ::placeholder {
+    color: ${(props) => props.theme.colors.textSecondary};
+`;
+
+const SimpleDataField = () => {
```

Comment:
This sounds very generic, I would name this component differently. Maybe: `MemberInfoInput`, `MemberDetailInput` or something that indicates a group of components: 'EditMemberDetail`

Reply:
MemberInfoInput sounds good. I will change it.